tests/pipelines/kitctl/run.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipeline-template
+  namespace: tekton-pipelines
+spec:
+  params:
+  - name: name
+    value: "example"
+  - name: substrate-name
```

Comment:
@prateekgogia can you help with naming

Reply:
We are going to use `management cluster` instead.